app/assets/stylesheets/content/_forms.sass

```diff
@@ -57,6 +57,8 @@ $form--field-types: (text-field, text-area, select, check-box, radio-button, ran
     padding: 30px 20px
     background-color: $content-form-bg-color
     border: $content-form-border
+  &.-higher
+    min-height: 31em
```

Comment:
I'm not familiar too with all frontend staff. This is the only solution that I've found and works. So far wasn't any other working ideas comes from other. So if you have some more appropriate approach I'm glad to see them.